docs/design_proposals/design-proposal-template.md

```diff
@@ -0,0 +1,83 @@
+# Title
+
+* Author(s): \<your name\>
+* Design Shepherd: \<skaffold-core-team-member\>
+
+    If you are already working with someone mention their name.
+    If not, please leave this empty, it will be assigned to a core team member.
+* Date: \<date\>
+* Status: [Draft/Reviewed/Complete]
```

Comment:
I think Status should be Reviewed / Under implementation / Complete / Cancelled - as the Draft phase is implicit while it's in the PR phase. We should also explain what these are ,e.g.:

`Reviewed` means that the proposal PR has been merged 
`Under implementation` means that the plans is being executed by actual work
`Complete` means that there is no changes expected in this design anymore, if 
`Cancelled` means that we learned something during or before implementation that caused this proposal to be cancelled. 

I would add a `Links` section as well, to collect relevant PRs and issues too. 

What do you think?

Reply:
> I would add a Links section as well, to collect relevant PRs and issues too.

I think we have mentioned that in the `Background` bullet 4.

Reply:
> I think Status should be Reviewed / Under implementation / Complete / Cancelled - as the Draft phase is implicit while it's in the PR phase. We should also explain what these are ,e.g.:

So @balopat along your lines but a little different, 

1. In Review: The proposal PR has not been reviewed yet by the core team.
2. Accepted: The proposal PR has been accepted and ready for implementation.
3. Not Accepted: The proposal PR was not accepted by the core team.
4. Under implementation: An accepted proposal is being implemented by actual work.
5. Cancelled: During or before implementation the proposal was cancelled.
   It could be due to:

   * other features added which made the current design proposal obselete.
   * no longer a priority.

6. Complete: This feature/change is implemented.